src/Illuminate/Support/Facades/Exceptions.php

```diff
@@ -25,16 +25,16 @@
  * @method static \Illuminate\Foundation\Exceptions\Handler shouldRenderJsonWhen(callable $callback)
  * @method static \Illuminate\Foundation\Exceptions\Handler dontReportDuplicates()
  * @method static \Illuminate\Contracts\Debug\ExceptionHandler handler()
- * @method static void assertNothingReported()
  * @method static void assertReported(\Closure|string $exception)
  * @method static void assertReportedCount(int $count)
  * @method static void assertNotReported(\Closure|string $exception)
+ * @method static void assertNothingReported()
  * @method static void renderForConsole(\Symfony\Component\Console\Output\OutputInterface $output, \Throwable $e)
+ * @method static \Illuminate\Support\Testing\Fakes\ExceptionHandlerFake throwOnReport()
  * @method static \Illuminate\Support\Testing\Fakes\ExceptionHandlerFake throwFirstReported()
  * @method static \Illuminate\Support\Testing\Fakes\ExceptionHandlerFake setHandler(\Illuminate\Contracts\Debug\ExceptionHandler $handler)
  *
  * @see \Illuminate\Foundation\Exceptions\Handler
- * @see \Illuminate\Contracts\Debug\ExceptionHandler
  * @see \Illuminate\Support\Testing\Fakes\ExceptionHandlerFake
  */
```

Comment:
We do not need the contract here. The `handler` already implements the contract.